lib/Sapi.php

```diff
@@ -71,6 +71,11 @@ static function sendResponse(ResponseInterface $response) {
         $body = $response->getBody();
         if (is_null($body)) return;
 
+        if (is_callable($body)) {
+            $body();
```

Comment:
looking at your generateMultiStatus example tells me that this assumption doesnt work?